addons/project/static/src/js/widgets/status_with_color_widget.js

```diff
@@ -17,6 +17,7 @@ export const StatusWithColor = FieldSelection.extend({
     init: function () {
         this._super.apply(this, arguments);
         this.color = this.recordData[this.nodeOptions.color_field];
+        this.hideLabel = this.nodeOptions.hide_label;
```

Comment:
Why not using the `nolabel` attribute? 🤔

Reply:
I did use `nolabel` to hide the column label, but also created this new option to hide the status text and use it as hover text on the status color dot instead (I chose the name `hide_label` to be consistent with the widget `state_selection`)

Reply:
Okay thanks. 🙂